package.json

```diff
@@ -94,6 +94,7 @@
     "@meteorwallet/sdk": "^1.0.9",
     "@mintbase-js/wallet": "0.6.0-beta.3",
     "@near-snap/sdk": "^0.6.0",
+    "@near-wallet-selector/modal-ui": "^8.9.10",
```

Comment:
I think we don't need to add @near-wallet-selector/modal-ui as dependency because of NX.